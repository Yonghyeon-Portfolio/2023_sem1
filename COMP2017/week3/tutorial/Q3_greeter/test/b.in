personC 53 Candle
