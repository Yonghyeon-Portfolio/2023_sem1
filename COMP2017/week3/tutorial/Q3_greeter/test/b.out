Welcome to ShopaMocha,
Customer  0, Name: personC, Age: 53, Looking for: Candle
